components/AppLayout/Widgets/Avatar.js

```diff
@@ -72,24 +91,96 @@ const StatusBadge = withStyles(theme => ({
   );
 });
 
+const OpenPeepsAvatar = withStyles(theme => ({
+  showcaseWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    height: '-webkit-fill-available',
+    cursor: 'pointer',
+    '& div': {
+      width: ({ size }) => size,
+      height: ({ size }) => size,
+      [theme.breakpoints.up('md')]: {
+        width: ({ size, mdSize }) => mdSize ?? size,
+        height: ({ size, mdSize }) => mdSize ?? size,
+      },
+      backgroundColor: getBackgroundColor,
+    },
+    '& svg': {
+      width: ({ size }) => size,
+      height: ({ size }) => size,
+      overflow: 'visible',
+      [theme.breakpoints.up('md')]: {
+        width: ({ size, mdSize }) => mdSize ?? size,
+        height: ({ size, mdSize }) => mdSize ?? size,
+      },
+
+      transform: ({ avatarData, size }) =>
+        `${
+          avatarData?.flip ? 'scale(-1, 1)' : 'scale(1, 1)'
+        } translateY(${size / 15}px)`,
+    },
+  },
+  // eslint-disable-next-line no-unused-vars
+}))(({ className, classes, avatarData, size, mdSize, ...rest }) => {
+  return (
+    <div className={className} {...rest}>
+      <div className={classes.showcaseWrapper}>
+        <Peep
```

Comment:
This will create 3 levels of div around the `svg`.
The inner most div is created by `circleStyle` in `Peep` (relevant code: https://github.com/CeamKrier/react-peeps/blob/master/src/peeps/index.tsx#L160 )

I think we can
1. Just wrap 1 div around `<Peep>`
2. put styles in `peepsStyles.circleStyle` to `classes.showcaseWrapper`.
3. remove `circleStyle` prop from `<Peep>` to prevent `react-peeps` from generating extra div that